independent-projects/tools/codestarts/README.adoc

```diff
@@ -29,6 +29,7 @@ mvn io.quarkus:quarkus-maven-plugin:create
 jbang cli@quarkusio create
 ----
 
+To disambiguate, we decided that, Codestart is the tool (the code generator system), a codestart is an item used by the tool, an Extension codestart is example code for an extension.
```

Comment:
This sentence is confusing. 

Codestart with an initial upper case is the system and all lower case is an item used by the tool? It would be nice if that is explained more clearly here

Reply:
@gastaldi wdyt?
```suggestion
We decided that:

* "Codestart(s)" is the tool (the code generator system).
* A "codestart" is a directory following a specific struture containing a `codestart.yml` and templates. It is used by the tool in combinations with other codestarts to generate a project.
* An "Extension codestart" is a specific codestart providing example code for a Quarkus extension.
```

Reply:
It reads better, but I think the main issue here is that I'm afraid the word `codestart` is being overused 😃

Reply:
yeah I agree, but we couldn't settle on better with @maxandersen.. 

Problem is that I named everything codestarts thinking it would only be for Quarkus extensions and now it's too late 😂